contracts/crowdsale/validation/WhitelistedCrowdsale.sol

```diff
@@ -55,8 +20,8 @@ contract WhitelistedCrowdsale is Crowdsale, Ownable {
     address _beneficiary,
     uint256 _weiAmount
   )
-    internal
     isWhitelisted(_beneficiary)
+    internal
```

Comment:
according to http://solidity.readthedocs.io/en/v0.4.24/style-guide.html:
"The visibility modifier for a function should come before any custom modifiers.".
I've requested a solium rule for this: https://github.com/duaraghav8/Solium/issues/216